idlookup/idlookup.py

```diff
@@ -0,0 +1,76 @@
+import asyncio
+import os
+import re
+import urllib.parse
+
+from redbot.core import checks, commands, data_manager, Config
+
+import discord
+import tsutils
+
+try:
+    import re2 as re
```

Comment:
No need for fancy fast regex hacks.  You only need to do this when you have user-defined regexes, so can just import re here.